discovery/config/src/main/java/org/apache/aries/rsa/discovery/config/Activator.java

```diff
@@ -30,6 +31,13 @@
 import org.osgi.service.remoteserviceadmin.EndpointEventListener;
 import org.osgi.util.tracker.ServiceTracker;
 
+
+@Header(name = Constants.BUNDLE_ACTIVATOR, value = "${@class}")
+@org.osgi.annotation.bundle.Capability( //
```

Comment:
why do you leave comments at end of line?

Reply:
It preserves the layout when I reformat the class in Eclipse. I would be totally happy for a better way to preserve some of such formatting.

Reply:
As an apache committer you can use Intelij Idea. Maybe give it a try?
https://blog.jetbrains.com/blog/2019/05/30/jetbrains-supports-the-apache-software-foundation/

Reply:
For apache it would work but for my employer it always depends if they want to spend the money. So I stay with eclipse.